diff --git a/dist/commands/export-dynamic-plugin/backend-embed-as-code.cjs.js b/dist/commands/export-dynamic-plugin/backend-embed-as-code.cjs.js
index 6cb368177c3970cdaf821c77ddcfd84d75103486..695d6009a4e90b5b30fc0daadd5ed932ff1a7687 100644
--- a/dist/commands/export-dynamic-plugin/backend-embed-as-code.cjs.js
+++ b/dist/commands/export-dynamic-plugin/backend-embed-as-code.cjs.js
@@ -238,7 +238,7 @@ async function backend(roleInfo, opts) {
     const yarn = "yarn";
     const version = child_process.execSync(`${yarn} --version`).toString().trim();
     const yarnInstall = version.startsWith("1.") ? `${yarn} install --production${yarnLockExists ? " --frozen-lockfile" : ""}` : `${yarn} install${yarnLockExists ? " --immutable" : " --no-immutable"}`;
-    await tasks.Task.forCommand(yarnInstall, { cwd: target, optional: false });
+    await tasks.Task.forCommand('yarn install', { cwd: target, optional: false });
     await fs__default.default.remove(paths.paths.resolveTarget("dist-dynamic", ".yarn"));
   }
   await fs__default.default.remove(paths.paths.resolveTarget("dist"));
diff --git a/dist/commands/export-dynamic-plugin/backend-embed-as-dependencies.cjs.js b/dist/commands/export-dynamic-plugin/backend-embed-as-dependencies.cjs.js
index 89299c34e115f25da32f91689d037934a1f605f7..0c5acbc9f54bde8c395f53ed1c42df5149d7fce8 100644
--- a/dist/commands/export-dynamic-plugin/backend-embed-as-dependencies.cjs.js
+++ b/dist/commands/export-dynamic-plugin/backend-embed-as-dependencies.cjs.js
@@ -316,7 +316,7 @@ throw new Error(
     const logFile = "yarn-install.log";
     const redirect = `> ${logFile}`;
     const yarnInstall = yarnVersion.startsWith("1.") ? `${yarn} install --production${yarnLockExists ? " --frozen-lockfile" : ""} ${redirect}` : `${yarn} install${yarnLockExists ? " --immutable" : " --no-immutable"} ${redirect}`;
-    await tasks.Task.forCommand(yarnInstall, { cwd: target, optional: false });
+    await tasks.Task.forCommand('yarn install', { cwd: target, optional: false });
     await fs__namespace.remove(paths.paths.resolveTarget(targetRelativePath, ".yarn"));
     tasks.Task.log(`Validating private dependencies`);
     const lockFile = await Lockfile.Lockfile.load(yarnLock);
